### PR TITLE
[zh-cn]: update the translation of the `cookie.Cookie` type

### PR DESCRIPTION
**Pull request by tianyeeT**
### Related issues and pull requests
#20967

### CODE REVIEW

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md [hunk 1]
```diff
@@ -1,50 +1,56 @@
...
 ## 类型
 
-这玩意是一个 Object，可以包含以下的属性：
+此类型的值是对象，可以包含以下属性：
 
-- `domain`
-  - : 储存这个 cookie 对应网站的字符串 (例如 "www\.tengxun.com")。
-- `expirationDate`{{optional_inline}}
-  - : A `number` representing the expiration date of the cookie as the number of seconds since the UNIX epoch. Not provided for session cookies.
+- domain
```
Comment (PassionPenguin) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md:16]:
```suggestion
- `domain`
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md [hunk 2]
```diff
@@ -1,50 +1,56 @@
...
+  - : 表示 cookie 所属域的 `string`（例如“www.google.com”，“example.com”）。
```
Comment (PassionPenguin) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md:17]:
```suggestion
  - : 表示 cookie 所属域的 `string`（例如“www.google.com”、“example.com”）。
```

Comment (PassionPenguin) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md:17]:
```suggestion
  - : 表示 cookie 所属域的 `string`（例如“www.google.com”、“example.com”）。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md [hunk 3]
```diff
@@ -1,50 +1,56 @@
...
+- `expirationDate` {{optional_inline}}
+  - : 表示 cookie 到期日期的 `number`，以 UNIX 纪元以来的秒数表示。会话 cookie 不提供此属性。
 - `firstPartyDomain`
-  - : A `string` representing the first-party domain associated with the cookie. This will be an empty string if the cookie was set while first-party isolation was off. See [First-party isolation](/zh-CN/Add-ons/WebExtensions/API/cookies#First-party_isolation).
+  - : 表示与 cookie 关联的第一方域的 `string`。如果在第一方隔离关闭时设置 cookie，此属性将为空字符串。参见 [First-party isolation](/zh-CN/docs/Mozilla/Add-ons/WebExtensions/API/cookies#first-party_isolation)。
```
Comment (PassionPenguin) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md:21]: 链接名

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md [hunk 4]
```diff
@@ -1,50 +1,56 @@
...
-  - : A `boolean`, `true` if the cookie is a host-only cookie (i.e. the request's host must exactly match the domain of the cookie), or `false` otherwise.
+  - : 一个 `boolean`，如果 cookie 仅适用于主机（即请求的主机必须与 cookie 的域完全匹配），则为 `true`，否则为 `false`。
 - `httpOnly`
-  - : A `boolean`, `true` if the cookie is marked as HttpOnly (i.e. the cookie is inaccessible to client-side scripts), or `false` otherwise.
+  - : 一个 `boolean`，如果 cookie 标记为 HttpOnly（即客户端脚本无法访问 cookie），则为 `true`，否则为 `false`。
 - `name`
-  - : A `string` representing the name of the cookie.
+  - : 表示 cookie 名称的 `string`。
+- `partitionKey` {{optional_inline}}
+  - : 一个 `object`，表示包含 cookie 的存储分区的描述。如果 cookie 不在分区存储中，则此对象省略（null）。此对象包含以下属性：
```
Comment (PassionPenguin) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md:29]: 原文没用backtick包围，就作为名词直接翻译吧

```suggestion
  - : 一个 `object`，表示包含 cookie 的存储分区的描述。如果 cookie 不在分区存储中，则此对象省略（空值）。此对象包含以下属性：
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md [hunk 5]
```diff
@@ -1,50 +1,56 @@
...
+    - `topLevelSite`
+      - : 如果 cookie 存储在按顶级站点分区的存储中，则表示 cookie 存储分区的第一方 URL 的 `string`。
 - `path`
-  - : A `string` representing the path of the cookie.
+  - : 表示 cookie 路径的 `string`。
 - `secure`
-  - : A `boolean`, `true` if the cookie is marked as secure (i.e. its scope is limited to secure channels, typically HTTPS), or `false` otherwise.
+  - : 一个 `boolean`，如果 cookie 标记为安全（即其范围仅限于安全通道，通常是 HTTPS），则为 `true`，否则为 `false`。
```
Comment (PassionPenguin) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md:35]:
```suggestion
  - : 一个 `boolean`，如果 cookie 标记为安全（即其作用域仅限于安全通道，通常是 HTTPS）则为 `true`，否则为 `false`。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md [hunk 6]
```diff
@@ -1,50 +1,56 @@
...
 - `session`
-  - : A `boolean`, `true` if the cookie is a session cookie, or `false` if it is a persistent cookie with an expiration date.
+  - : 一个 `boolean`，如果 cookie 是会话 cookie，则为 `true`；如果是具有到期日期的持久 cookie，则为 `false`。
 - `sameSite`
-  - : A {{WebExtAPIRef("cookies.SameSiteStatus")}} value that indicates the SameSite state of the cookie.
+  - : 一个 {{WebExtAPIRef("cookies.SameSiteStatus")}} 值，指示 cookie 的 SameSite 状态。
```
Comment (PassionPenguin) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md:39]:
```suggestion
  - : 一个 {{WebExtAPIRef("cookies.SameSiteStatus")}} 值，指示 cookie 的同站状态。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md [hunk 7]
```diff
@@ -1,50 +1,56 @@
...
-## Browser compatibility
+## 浏览器兼容性
 
 {{Compat}}
 
-## 举例
+## 示例
 
-Cookies API 中的大多数方法都将 `Cookie` 对象用作输入参数或用作返回值的一部分。例如调用 {{WebExtAPIRef("cookies.getAll()")}} 将会返回一个 `Cookie` 对象的数组。
+cookies API 中的大多数方法涉及使用 `Cookie` 对象作为输入参数或作为返回值的一部分。例如，调用 {{WebExtAPIRef("cookies.getAll()")}} 会返回一个 Cookie 对象数组。
```
Comment (PassionPenguin) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md:51]:
```suggestion
cookies API 中的大多数方法涉及使用 `Cookie` 对象作为输入参数或作为返回值的一部分。例如，调用 {{WebExtAPIRef("cookies.getAll()")}} 会返回一个 `Cookie` 对象数组。
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md [hunk 8]
```diff
@@ -1,50 +1,56 @@
...
 
-在下面的例子中我们将会获取所有的 cookie，然后 `console.log()` 出这些 `Cookie` 对象所对应的值。
+在下面的示例中，我们请求了所有的 cookies，然后记录了每个 `Cookie` 对象的一些值：
```
Comment (PassionPenguin) [inline files/zh-cn/mozilla/add-ons/webextensions/api/cookies/cookie/index.md:53]:
```suggestion
在下面的示例中，我们请求获取所有的 cookie，然后输出每个 `Cookie` 对象的一些值：
```

**Review by PassionPenguin — CHANGES_REQUESTED** [review]

**Comment by tianyeeT** [comment]
Hi @PassionPenguin, thanks for reviewing this PR. All the issues mentioned above have been addressed.

**Review by yin1999 — APPROVED** [review]

**Pull request merged**